src/amo/components/AddonTitle/index.js

```diff
@@ -56,35 +58,37 @@ export const AddonTitleBase = ({
     });
   }
 
-  return (
-    <h1 className="AddonTitle">
```

Comment:
Quick question: why not declaring a prop like `component` that defaults to `h1` and use it here?

Reply:
yeh let me try that... thanks!